roles/custom/matrix-bridge-hookshot/templates/systemd/matrix-hookshot.service.j2

```diff
@@ -30,11 +28,18 @@ ExecStart={{ devture_systemd_docker_base_host_command_docker }} run --rm --name
           {% endfor %}
           {{ matrix_hookshot_docker_image }}
 
-ExecStop=-{{ devture_systemd_docker_base_host_command_docker }} stop --time={{ devture_systemd_docker_base_container_stop_grace_time_seconds }} {{ matrix_hookshot_container_url }}
-ExecStop=-{{ devture_systemd_docker_base_host_command_docker }} rm {{ matrix_hookshot_container_url }}
+{% if matrix_hookshot_container_additional_networks %}
+{% for network in matrix_hookshot_container_additional_networks %}ExecStartPre={{ devture_systemd_docker_base_host_command_docker }} network connect {{ network ]] {{ matrix_hookshot_container_ident }}{% endfor %}
+{% endif %}
```

Comment:
```suggestion
{% for network in matrix_hookshot_container_additional_networks %}ExecStartPre={{ devture_systemd_docker_base_host_command_docker }} network connect {{ network ]] {{ matrix_hookshot_container_ident }}{% endfor %}
```

It's simpler to not have the `{% if .. %}`.

As discussed before, all roles deal with variables that must be defined and have some default value in `defaults/main.yml`. I don't see `matrix_hookshot_container_additional_networks` defined yet.

Another review comment of mine discussed adjusting `group_vars/matrix_servers` to re-define `matrix_hookshot_container_additional_networks_auto`, so let's prepare these variables and their defaults by adding this to `defaults/main.yml`:

```yml
# Specifies additional networks that the Hookshot container would be connected to
matrix_hookshot_container_additional_networks: "{{ matrix_hookshot_container_additional_networks_auto + matrix_hookshot_container_additional_networks_custom }}"
matrix_hookshot_container_additional_networks_auto: []
matrix_hookshot_container_additional_networks_custom: []
```

You can add this new block somewhere around the other `matrix_hookshot_container_*` variables in `defaults/main.yml`

Reply:
thanks